lib/api/assertions/_assertionInstance.js

```diff
@@ -80,6 +80,10 @@ class AssertionInstance {
     return this.__negate;
   }
 
+  get retryAssertionTimeout() {
+    return Array.isArray(this.__args) && this.__args[0].timeout;
```

Comment:
there's a getter for this.args, so you can rewrite this as: return this.args[0]

Reply:
done